src/Assert.php

```diff
@@ -968,14 +970,16 @@ public static function oneOf($value, array $values, $message = '')
     }
 
     /**
-     * @param mixed  $value
+     * @param string $value
```

Comment:
Im afraid this is a BC break. 

Especially when we add `static::string($value, $message);`.

Now we have to pass a string or an exception is thrown. Before we could pass something that could cast to string. 

See https://3v4l.org/ptW4Y

Reply:
Didn't know strpos allowed that.

Would changing the doc block also be a BC break?